src/main/java/com/shippingcostestimator/enterprise/dto/Shipment.java

```diff
@@ -54,11 +54,8 @@
     //private Map<String, Object> fromAddress = new HashMap<String, Object>();
 
     //PARCEL
-    private double length;
-    private double width;
-    private double height;
-    private String predefinedPackage;
-    private double weight;
+    private PackageInfo packageInfo;
+
```

Comment:
Since I gave the package its own class. Here in the shipment class, you will pass the package back into the shipment if need be. You can always turn this into a list array of packages to hold more if you wish to calculate multiple packages in the future.

Reply:
Good idea.  Separate the concept of shipment and package.